parsers/RU.py

```diff
@@ -50,196 +66,94 @@
     "UA->RU-1": 880,
 }
 
+SOURCE = "so-ups.ru"
+
 # Each exchange is contained in a div tag with a "data-id" attribute that is unique.
 
 
 TIMEZONE = ZoneInfo("Europe/Moscow")
 
 
 def fetch_production(
-    zone_key: str = "RU",
+    zone_key: ZoneKey,
     session: Session | None = None,
     target_datetime: datetime | None = None,
     logger: Logger = getLogger(__name__),
 ) -> list[dict] | dict:
-    """Requests the last known production mix (in MW) of a given country."""
-    if zone_key == "RU":
```

Comment:
There is no point in this as we will use the subzone data and aggregate it in the backend instead.